client/post-editor/post-editor.jsx

```diff
@@ -713,9 +713,6 @@ export const PostEditor = React.createClass( {
 			this.props.resetEditorLastDraft();
 		}
 
-		// Remove this when the editor is completely reduxified ( When using Redux actions for all post saving requests )
-		this.props.savePostSuccess( post.site_ID, this.props.postId, post, {} );
```

Comment:
If we remove this, we'll want to remove the prop from `mapDispatchToProps` since this is the only place it's used. Just a small observation, still doing a more in-depth review...